### ISSUE TITLE
Google Fonts are not loaded in the front-end

### ISSUE THREAD

**Issue by roljohntorralba** [themeum/kirki#2249]
### Issue description:
Hello, I'm using the Typography field, everything is working properly in the Customize preview, but when the changes are published and when viewed in the front-end, the font is non-existent.

All I can find in the page source is the embedded CSS that sets the element's typography properties, no enqueued google fonts, and no styles loading the font face.

I was first developing the theme in my localhost (xampp), and I thought this is a localhost configuration bug, so I created a demo website online (under a subdomain) and uploaded my theme on a fresh install of WordPress, but I get the same result --works in customizer preview, but not in the front-end.

I also think this issue and one issue I submitted earlier (#2248 ) was caused by something in my configuration of Kirki, embedded inside the theme.

Thanks and thank you for making this great library.

### Version used:
Kirki: 3.0.44
WordPress: 5.2.2
PHP: 7.1

### Using theme_mods or options?
theme_mods only

### Code to reproduce the issue (config + field(s))
Embedded in the theme.

**Topmost code in my `functions.php`**
```
/**
 * Load Kirki
 */
require_once get_template_directory() . '/kirki-develop/kirki.php';
```

**Topmost code in my `customizer.php`**
```
use Kirki\Util\Helper;

// Exit if accessed directly.
if ( ! defined( 'ABSPATH' ) ) {
	exit;
}

// Do not proceed if Kirki does not exist.
if ( ! class_exists( 'Kirki' ) ) {
	return;
}
```

**Somewhere below the `customizer.php`**
```
// Body Font
Kirki::add_field( 'wsubusiness_kirki', array(
	'type'        => 'typography',
	'settings'    => 'typography_body',
	'label'       => esc_html__( 'Body Font', 'kirki' ),
	'section'     => 'typography_options',
	'default'     => array(
		'font-family'    => '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif',
		'variant'        => 'regular',
		'line-height'    => '1.685',
		'text-transform' => 'none',
	),
	'priority'    => 10,
	'choices' => array(
		'fonts' => array(
			'google'   => array( 'popularity', 50 ),
			'standard' => array(
				'Georgia,Times,"Times New Roman",serif',
				'-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif',
			),
		),
	),
	'transport'   => 'auto',
	'output'      => array(
		array(
			'element' => 'body',
		),
	),
) );
```

**Comment by aristath** [comment]
A couple of questions:
Is this for a theme you're currently developing and want it to go live? Or are you simply testing the develop branch? 
From what I can tell from the existence of the `Kirki\Util\Helper` reference in your code you're using the develop branch which is a WIP rewrite of the plugin for the 4.0 version.
Was that on purpose? If not, please use the stable version of the plugin instead, you can download it from w.org/plugins/kirki

If this was on purpose then things change and you have my gratitude for reporting this issue for v4.0 😄

**Comment by roljohntorralba** [comment]
Honestly, that was a stupid mistake. I can't recall where I downloaded that develop branch package, but installing the stable version fixed everything including my #2248 issue.

Thanks a ton!
